### PR TITLE
Create ui-c8y-1020-14-6-device-owner-toggle-will-not-switch-back-after-cancelling.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3806](https://cumulocity.atlassian.net/browse/DM-3806)
Original PR: [6827](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6827)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-14-6-device-owner-toggle-will-not-switch-back-after-cancelling.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Device owner toggle will not switch back after cancelling
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-14-6-device-owner-toggle-will-not-switch-back-after-cancelling.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Device owner toggle no longer remains in incorrect state after canceling the change
```

File: content/change-logs/device-management/ui-c8y-1020-14-6-device-owner-toggle-will-not-switch-back-after-cancelling.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3806
+version: 1020.14.6
+---
+Device owner toggle will not switch back after cancelling
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-14-6-device-owner-toggle-will-not-switch-back-after-cancelling.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the device details view, administrators can assign the device owner. However, when attempting to change the device owner and then canceling the action, the toggle control previously remained in an incorrect state and appeared as if the change had been applied. With this fix, canceling the device owner change now correctly reverts the toggle control back to the original state, avoiding confusion about the current device owner assignment.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1020-14-6-device-owner-toggle-will-not-switch-back-after-cancelling.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1020-14-6-device-owner-toggle-will-not-switch-back-after-cancelling.md:17]:
```suggestion
In Device management, when attempting to change the device owner on the device "Info" tab and then canceling the action, the toggle control previously remained in an incorrect state and appeared as if the change had been applied. With this fix, canceling the device owner change now correctly reverts the toggle control back to the original state, avoiding confusion about the current device owner assignment.

```

File: content/change-logs/device-management/ui-c8y-1020-14-6-device-owner-toggle-will-not-switch-back-after-cancelling.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3806
+version: 1020.14.6
+---
+In Device management, when attempting to change the device owner on the device "Info" tab and then canceling the action, the toggle control previously remained in an incorrect state and appeared as if the change had been applied. With this fix, canceling the device owner change now correctly reverts the toggle control back to the original state, avoiding confusion about the current device owner assignment.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-14-6-device-owner-toggle-will-not-switch-back-after-cancelling.md:17]:
```suggestion
When attempting to change the device owner on the **Info** tab in the device details and then canceling the action, the toggle control previously remained in an incorrect state and appeared as if the change had been applied. With this fix, canceling the device owner change now correctly reverts the toggle control back to the original state, avoiding confusion about the current device owner assignment.
```

**Review by BeateRixen — APPROVED** [review]

**Review by dafkk — APPROVED** [review]

**Pull request merged**